app/src/ui/lib/fancy-text-box.tsx

```diff
@@ -38,16 +38,9 @@ export class FancyTextBox extends React.Component<
       <div className={componentCSS}>
         <Octicon className={octiconCSS} symbol={this.props.symbol} />
         <TextBox
-          value={this.props.value}
+          {...this.props}
```

Comment:
Is the new pattern we're going to adopt?

Reply:
I'd say this is an exceptional case. Initially I added a new prop to `TextBox` and because `FancyTextBox`'s props extends the props for `TextBox` I thought it'd just work, but it didn't because we were passing in each prop explicitly and the new one was omitted. 

So after confirming explicitly passing the new prop in I went back and cleaned this up so that it wouldn't happen to the next person touching this code. I think we agreed that `FancyTextBox` isn't a long-term thing, so I don't want to encourage this either unless we find specific cases like this that will prevent mistakes.

Alternatively, I can revert this cleanup code. Let me know if that's a more conservative change in the context of this PR.

Reply:
Reverted :boom: